tsconfig.json

```diff
@@ -17,6 +17,6 @@
         "forceConsistentCasingInFileNames": true,
         "types": ["vite/client"]
     },
-    "include": ["src"],
-    "exclude": ["**/*.test.ts", "node_modules", "test/**", ".history/**"]
+    "include": ["src", "test"],
```

Comment:
We want TS running on our tests, right? 🤔 We can't lint them (with rules that requires types) if they aren't included here.

Reply:
Well, this broke the build so maybe not? 😅

Reply:
Seems legit? As long as they don't get included in the bundle right?

Reply:
Maybe we need a different tsconfig for linting purposes?

Reply:
Added a separate tsconfig for tests that don't try to emit types.